pkg/grapicmd/context.go

```diff
@@ -0,0 +1,76 @@
+package grapicmd
+
+import (
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
+
+	"github.com/izumin5210/grapi/pkg/grapicmd/protoc"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+)
+
+type Ctx struct {
+	FS        afero.Fs
+	Viper     *viper.Viper
+	InReader  io.Reader
+	OutWriter io.Writer
+	ErrWriter io.Writer
+
+	CurrentDir string
+	RootDir    string
+	InsideApp  bool
+
+	AppName   string
+	Version   string
+	Revision  string
+	BuildDate string
+	Prebuilt  bool
+
+	Config       Config
+	ProtocConfig protoc.Config
+}
+
+// Config stores general setting params and provides accessors for them.
+type Config struct {
+	ServerDir string
+	Package   string
+}
+
+func (c *Ctx) Init(cfgFile string) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Ctx.Init should have comment or be unexported